utils.py

```diff
@@ -157,27 +159,25 @@ def time(cmd):
 def weather(cmd):
 	if not cmd.args:
 		return
-	split = cmd.args.split()
-	if split[0].startswith('-'):
-		flags = split[0][1:]
-		location = ' '.join(split[1:])
-	elif split[-1].startswith('-'):
-		flags = split[-1][1:]
-		location = ' '.join(split[:-1])
-	else:
-		flags = '1Fp'
-		location = cmd.args
-		if location.isdecimal() and len(location) == 5:
-			location += '-us'
-	url = 'https://wttr.in/%s.png?%s' % (urllib.parse.quote_plus(location), flags)
+	flags = 'format=**%l:**+%c+++🌡+`%t(%f)`++💦+`%h`++💨+`%w`++**☔**+`%p/3h`++**UVI:**+`%u`\
+				\n**Time:**+`%T`++**Sunrise:**+`%S`++**Sunset:**+`%s`++**Moon:**+%m'
```

Comment:
I... wasn't aware this was allowed. and it just dedents the whitespace on the second line or something?
can we uh
```suggestion
	flags = 'format=**%l:**+%c+++🌡+`%t(%f)`++💦+`%h`++💨+`%w`++**☔**+`%p/3h`++**UVI:**+`%u`\n' \
			'**Time:**+`%T`++**Sunrise:**+`%S`++**Sunset:**+`%s`++**Moon:**+%m'
```

Reply:
https://github.com/raylu/sbot/actions/runs/8520604984/job/23337068220?pr=23#step:6:16
Linter does not seem to like this